docs/docs/gpus.md

```diff
@@ -0,0 +1,69 @@
+# Preferential GPU Scheduling
+
+## Overview of GPU Scheduling in Metronome
+
+In order to have Metronome run jobs with GPU workloads it is necessary to:
+
+1. Understand the Mesos configuration for GPU management.
+2. Start Metronome with GPU behavior (`METRONOME_GPU_SCHEDULING_BEHAVIOR` detailed below).
+3. Specify gpus resource using whole numbers only.
+4. Use the UCR containerizer.
+
+
+Assuming that Mesos is configuration to filter GPU resources (details below), the following is what is necessary in metronome.
+
+## GPU Startup Parameters in Metronome
+
+Metronome supports launching GPU tasks when the environmental variable `METRONOME_GPU_SCHEDULING_BEHAVIOR` is specified.  If this env var is *not* specified, Metronome will not opt-in for receiving Mesos offers which contain GPU resources.  Depending on the GPU availability, it may be desirable to configure Metronome to avoid placing non-GPU tasks on GPU nodes. By *not* specifying GPU scheduling *and* while Mesos is configured to filter GPU resources, Metronome will *not* receive offers containing GPU resources.   When `METRONOME_GPU_SCHEDULING_BEHAVIOR` is specified with an appropriate option (listed below), then Metronome is opting in to receiving offers with GPUs.  Below is the options and behavior when specifying Metronome GPU scheduling behavior via the `METRONOME_GPU_SCHEDULING_BEHAVIOR` env variable when starting Metronome:
```

Comment:
s/environmental variable/environment variable/

Reply:
Inconsistent use of environment variable, env variable, env var

Reply:
yeah.. thanks... fixed and made consistent with environment variable